tensorflow_model_optimization/g3doc/guide/clustering/clustering_with_keras.ipynb

```diff
@@ -0,0 +1,1164 @@
+{
```

Comment:
<p>Generally, if you have any questions on differences between the old (existing pruning docs on tensorflow.org) and new (QAT and WIP pruning), let me know. It's important to understand the motivation both to save time and also make future edits easier. You're even welcome to directly comment on the pending QAT PRs for questions. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>For the end-to-end example: <a href="https://github.com/tensorflow/model-optimization/pull/289" target="_blank">here</a></p><p>For the comprehensive guide: <a href="https://github.com/tensorflow/model-optimization/pull/290" target="_blank">here</a></p>